loggingpy/log.py

```diff
@@ -268,13 +268,15 @@ def to_dict(self, obj, classkey=None):
     def format(self, record):
         """Formats a log record and serializes to json"""
 
+        record.msg = logging.Formatter.format(record) # format the message using the base formatter
```

Comment:
Can we use a formatter that leaves out all the verbose stuff like the duplicate timestamp and level?

Reply:
I wouldn't know about any of that :)
Basically, I'm not suggesting how to do something - I just noticed in the logs that you are logging strings with the "default" format, which contain the levels and timestamps. Wouldn't it make sense to override this with a custom formatter (unless the developer already provided a custom formatter of their own)?